docs/developers_guide/StreamingLiteComponents.md

```diff
@@ -24,13 +24,21 @@ or `NuExceptionInfo` in case of an error.
 
 ## Sources and sinks
 
+### Streaming
+
 In Streaming-Lite sources and sinks are relatively simple, as all sources and sinks are based on Kafka:
 - A [source](https://github.com/TouK/nussknacker/blob/staging/engine/lite/kafka/components-api/src/main/scala/pl/touk/nussknacker/engine/lite/kafka/api/LiteKafkaSource.scala) 
 transforms `ConsumerRecord` to Nussknacker [Context](https://github.com/TouK/nussknacker/blob/staging/components-api/src/main/scala/pl/touk/nussknacker/engine/api/Context.scala). 
 - A [sink](https://github.com/TouK/nussknacker/blob/staging/engine/lite/components-api/src/main/scala/pl/touk/nussknacker/engine/lite/api/customComponentTypes.scala#L51) creates `ProducerRecord` from its parameters.
 - Both may return error.
 
-## Lite custom transformers
+### Request-Response
+
+In Request-Response-Lite sources and sinks act slightly different. Both create a combination of a HTTP request (source) and a response(sink)
```

Comment:
We do not have a convention yet how to create a combination of _processing mode_ and _engine_. Seeing _Request-Response_Lite_ my concern is that it may not be clear what kind of creature this is. What about convention like this: 
_Processing mode / engine_ so we would end up with _Request-response / Lite_? This '/' suggests that we have 2  parts here, with '-' as a separator this looks kinda confusing to me.

Reply:
we can try with that approach